lib/components/navbar.vue

```diff
@@ -15,8 +15,11 @@
                     this.variant ? `bg-${this.variant}` : null,
                     this.fixed ? `fixed-${this.fixed}` : null,
                     this.sticky ? 'sticky-top' : null,
-                    this.toggleable ? `navbar-toggleable-${this.toggleBreakpoint}` : null
+                    this.toggleable ? this.toggleClass : null
                 ];
+            },
+            toggleClass() {
+                return ['navbar-toggleable' + this.toggleBreakpoint].join('-');
```

Comment:
Yeah, I mentioned that above. I put this together a little too hastily between today's hectic schedule.